packages/calendar/stories/index.stories.mdx

```diff
@@ -0,0 +1,257 @@
+  `}
+</Story>
+
+```html
+<lion-calendar .selectedDate="${new Date(1988, 2, 5)}"></lion-calendar>
+```
+
+## Central Date
+
+The `centralDate` defines which day will be focused when keyboard moves the focus to the current month grid.
+By default it is set to today, or the enabled day of the current month view that is closest to today's date.
+
+The next and previous months' buttons work by changing the `centralDate` with plus or minus one month.
+Changing the `centralDate` may mean a different view will be displayed to your users if it is in a different month.
+Usually if you change only the day, "nothing" happens as it's already currently in view.
+
+The `centralDate` can be different from `selectedDate` as you can have today as actively selected but still look at date that is years ago.
+When the `selectedDate` changes, it will sync its value to the `centralDate`.
+
+<Story name="Central Date">
+  {() => {
+    const today = new Date();
+    const centralDate = new Date(today.getFullYear(), today.getMonth() + 1, today.getDate());
+    return html`
+      <style>
+        ${calendarDemoStyle}
+      </style>
+      <lion-calendar class="demo-calendar" .centralDate="${centralDate}"></lion-calendar>
+    `;
+  }}
+</Story>
+
+```html
+<lion-calendar .centralDate="${new Date(1988, 2, 5)}"></lion-calendar>
+```
+
+## Controlling focus
+
+You can control the focus by calling the following methods
+- `focusCentralDate()`
+- `focusSelectedDate()`
+- `focusDate(dateInstanceToFocus)`
+
+Be aware that the central date changes when a new date is focused.
+
+<Story name="Controlling focus">
+  {() => {
+    const today = new Date();
+    const selectedDate = new Date(today.getFullYear(), today.getMonth(), today.getDate() + 1);
+    const centralDate = new Date(today.getFullYear(), today.getMonth() + 1, today.getDate());
+    return html`
+      <style>
+        ${calendarDemoStyle}
+      </style>
+      <lion-calendar
+        id="js-demo-calendar"
+        class="demo-calendar"
+        .selectedDate="${selectedDate}"
+        .centralDate="${centralDate}"
+      ></lion-calendar>
+      <p>
+        Focus
+        <button @click="${() => document.querySelector('#js-demo-calendar').focusCentralDate()}">
+          Central date
+        </button>
+        <button @click="${() => document.querySelector('#js-demo-calendar').focusSelectedDate()}">
+          Selected date
+        </button>
+        <button @click="${() => document.querySelector('#js-demo-calendar').focusDate(today)}">
+          Today
+        </button>
+      </p>
+    `;
+  }}
+</Story>
+
+## Limiting selectable values
+
+### Providing a lower limit
+
+To give a lower limit you can bind a date to the `minDate` property.
+
+<Story name="minDate">
+  {() => {
+    const minDate = new Date();
+    return html`
+      <style>
+        ${calendarDemoStyle}
+      </style>
+      <lion-calendar class="demo-calendar" .minDate="${minDate}"></lion-calendar>
```

Comment:
Do you think it would make sense to specify if `minDate` or `maxDate` specify the bounds of a closed or open interval? 
Given a `date X` and setting `minDate to Y` would dates be disabled up to `Y - 1` or up to `Y`?
Given a `date X` and setting `maxDate to Y` would dates be disable starting from `Y + 1` or from `Y`?